src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidUpdateResourcesTest.cs

```diff
@@ -87,7 +87,7 @@ public void DesignTimeBuild ([Values(false, true)] bool isRelease, [Values (fals
 					b.Target = "Compile";
 					Assert.IsTrue (b.Build (proj, doNotCleanupOnUpdate: true, parameters: new string [] { "DesignTimeBuild=true" }, environmentVariables: envVar),
 						"first build failed");
-					Assert.AreEqual (!useManagedParser, b.LastBuildOutput.Contains ("Skipping GetAdditionalResourcesFromAssemblies"),
+					Assert.AreEqual (!useManagedParser, b.LastBuildOutput.ContainsText ("Skipping GetAdditionalResourcesFromAssemblies"),
```

Comment:
Likely not relevant for *this* PR, but I half wonder if we should *kill* the `LastBuildOutput` property, and introduce a `LastBuildOutputContainsAny(params string[])` method. That way we could direct the output to a file, and only read into memory if it's deemed "small" enough.

Note the `params string[]`: this is so you can search for multiple desired strings at once, instead of separately:

    b.LastBuildOutputContainsAny ($"Downloading {url}", $"reusing existing archive: {zipPath}", $"{url} should have been downloaded.");